lib/src/widget/settings/data/settings_tile_action.dart

```diff
@@ -1,6 +1,22 @@
 import 'package:equatable/equatable.dart';
 import 'package:flutter/material.dart';
 
+enum SettingsTileActionTapArea {
+  content,
+  trailing,
+}
+
+extension SettingsTileActionExtension on SettingsTileAction {
```

Comment:
❓ why extension, but not setting value via constructor?

Reply:
It's something I prefer to do in Swift so thought it would be appropriate here as well 🙂

Reply:
My concern about extensions: its harder to test them 😞